src/App.js

```diff
@@ -10,49 +10,47 @@
 */
 
 import { Container, ListGroup, ListGroupItem, Navbar } from "react-bootstrap";
+import { useState, useEffect } from "react";
 
-async function getNewFact() {
+export default function App() {
+  const [facts, setFact] = useState([]);
+
+  async function getNewFact() {
     const response = await fetch("https://catfact.ninja/fact");
     const obj = await response.json();
-
-    return obj.fact;
-}
-
-export default function App() {
-    // Use state here
-    // Hint: https://reactjs.org/docs/hooks-state.html
-    // Hint: https://reactjs.org/docs/state-and-lifecycle.html#using-state-correctly
-    const facts = [
-        "Cats are weird",
-    ];
-
-    // Using the getNewFact function, load a new fact when this component is first loaded
-    // Hint: https://reactjs.org/docs/hooks-effect.html
-
-    return (
-        <Container>
-            <Navbar as="header" bg="primary" className="bg-gradient">
-                <Container fluid>
-                    <Navbar.Brand className="text-white">Tema React 2</Navbar.Brand>
-                </Container>
-            </Navbar>
-
-            <Container as="main" className="my-5 flex-grow-1">
-                <ListGroup>
-                    <ListGroup.Item className="fw-bold">Cat facts</ListGroup.Item>
-                    {facts.map((fact) => <ListGroupItem key={fact}>{ fact }</ListGroupItem>)}
-                    
-                    <ListGroup.Item action={true} variant="primary">
-                        {/* 
-                            On click, add a new fact to the list.
-
-                            Docs for List Group Item component: 
-                            https://react-bootstrap.netlify.app/components/list-group/#list-group-item-props
-                        */}
-                        Get new fact
-                    </ListGroup.Item>
-                </ListGroup>
-            </Container>
+    setFact([obj.fact]);
```

Comment:
Am putea să adăugăm la listă.

```suggestion
    setFact([...facts, obj.fact]);
```